src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const totalPrice = days * price;
+  const smallDiscont = 20;
+  const bigDiscount = 50;
+
+  if (days < 3 && days > 0) {
+    return totalPrice;
+  } else if (days >= 3 && days < 7) {
+    return totalPrice - smallDiscont;
+  }
```

Comment:
1. All else statements should be removed.
2. Find a way to use single conditions instead of double, e.g. `days >= 7`, `days >= 3` etc.